app/src/main/java/edu/mit/mitmobile2/facilities/FacilitiesManager.java

```diff
@@ -86,20 +86,16 @@ public static FacilityManagerCall getPlaceCategories(Activity activity, Callback
     }
 
     /* POST requests */
-
-    // http://mobile-dev.mit.edu/apis/building_services/
-    // http://m.mit.edu/apis/building_services/problems
-
-    // seems like it requires touchstone authorization, skip for now
-    public static void postProblem(String email, String message, String problemType, Callback<Response> callback) {
+    public static void postProblem(String email, String location, String room, String problem, String description, String photo, Callback<Response> callback) {
```

Comment:
I think Alex put this in here originally and I missed it -- We shouldn't be building an entirely new RestAdapter and Service since it's an expensive undertaking. Just use the static MIT_FACILITY_SERVICE at the top of the class file to make the POST request